docs/debug-agent.md

```diff
@@ -0,0 +1,41 @@
+# Get Clear Containers Agent debug logs
+
+When a new pod is created Clear Containers runtime launch a special VM which
+uses a VM-agent. The VM-agent allows to spawn processes on behalf of
+pod/container(s) running inside the virtual machine. 
+
+From Clear Containers 3.0-beta release a new agent is used,
+[Clear Containers Agent](https://github.com/clearcontainers/agent):
+
+The Clear Containers Agent  relies heavily on libcontainer used by runc
+(standard to run containers on bare metal).
+
+To provide a debug log of any agent activity on a guest, the Clear Containers
+agent sends logs through a qemu serial console` that are collected by `cc-proxy
+and shown in its logs. By default, this agent logs are not collected by 
+[cc-proxy](https://github.com/clearcontainers/proxy) but can be enabled with a
+simple change to service file.
+
+1- Add `-log debug` option to  `cc-proxy`  
+
+```
+$sudo sed -i \
```

Comment:
you could create a configuration file in /etc/systemd/system/cc-proxy.service.d to override the ExecStart line